test-interfaces/src/main/java/org/terracotta/passthrough/IClusterControl.java

```diff
@@ -54,7 +54,10 @@
    * If all servers are running, does nothing.
    *
    * @throws Exception Implementation-defined failure.
+   *
+   * Deprecated: same as @link #startOneServer.  consistency should be set in the startup
    */
+  @Deprecated()
   public void startOneServerWithConsistency() throws Exception;
```

Comment:
We could remove this this for good. The downstream repositories need to change several things to adapt to the latest galvan changes. They might as well fix their tests to use the new way.

Reply:
I considered that but seems pretty extreme.

Reply:
i looked at removing.  complete extraction is involved.  if we are in a hurry, let's deprecate for now.